### PR TITLE
Feature/static site

### PR DESCRIPTION
**Pull request by jnanic**
Only until 20 Nov. config files enroute,

### CODE REVIEW

**Comment by bogderp** [comment]
Won't get to this until wednesday night but just looking at it.

1. remove the testflasksite.tk directory. 
2. clean up your js and css. if you are using webpack to generate your js/css include any necessary files. 
3. don't put dynamic js/css under version control, generate them on your dev/prod server when deploying your site. 
4. upload your css/js assets that are being compiled into your compressed dynamic js/css
5. write a brief readme on how to generate dynamic assets

@HarrisonSanders @yash1337

File: README.md [hunk 1]
```diff
@@ -1 +1,44 @@
-# SpartaHackIV-Website
\ No newline at end of file
+# SpartaHack Static Site on Angular CLI
+****
+
+This is a guide for running a local copy of the SpartaHack Static Site at http://localhost:4200
+
+**Once set up, only Postgres.app and the rails server need to be on to run the site**
+    
+    rails server
```
Comment (bogderp) [inline README.md]: Do you really need this?

Comment (nathankurt) [reply README.md]: I mean I think the http://localhost:4200 thing is helpful right? isn't it normally localhost:5000?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
+{
+  "name": "sparta-hack",
```
Comment (bogderp) [inline package.json]: nit: sparta-hack makes me cringe. this should be spartahack

File: README.md (hunk 1 above)
Comment (bogderp) [reply README.md]: I'm referring to rails. I don't think angular has a rails dependency

Comment (bogderp) [reply README.md]: for reference, i place my comments on the offending line

Comment (HavenTheWanderer) [reply README.md]: Yeah, I copy-pasted an old readme and missed that, sorry!

File: src/app/app.component.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<div [@fadeIn]=''>
+    <app-head></app-head>
+</div>
+<app-top></app-top>
+<div [@fadeIn]=''>
+    <app-faq></app-faq>
```
Comment (bogderp) [inline src/app/app.component.html:6]: just a comment, do you need to have all your components start with `app-`. could you not rather have the prefix be page specific or omit it all together?

Comment (HavenTheWanderer) [reply src/app/app.component.html:6]: It's an Angular convention. As an app gets more complex I usually abandon it, but I use it to denote the stuff that is on the very top

File: src/app/header/header.component.html [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+			</a>
+		</div>
+		<div class="medium-2 small-3 columns menu-button">
+			<a scrollTo href="#mail">
+				Sign Up!
+			</a>
+		</div>
+		<div class="medium-2 small-3 columns menu-button">
+			<a scrollTo href="#faq">
+				FAQ's
```
Comment (bogderp) [inline src/app/header/header.component.html]: faq stands for frequently asked questions so the `'s` is unnecessary

File: src/app/services/faq.service.ts [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    // 'content-type': 'application/json',
+    // 'authorization': 'Token token=9ba5cc744d40feb58d0bc6bce7c8d8be',
+    // 'accept': 'vnd.example.v2'
+    });
+
+  constructor(private http: Http) { }
+
+  getFaqs(): Observable<Faq[]> {
+    let options = new RequestOptions({ headers : this.headers});
+    return this.http.get('https://api.elephant.spartahack.com/faqs', options)
```
Comment (bogderp) [inline src/app/services/faq.service.ts:30]: you should probably take this from an environment file or config. Your production server will not be this endpoint

File: src/app/services/sponsor.service.ts [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+  logoURL: string;
+}
+
+@Injectable()
+
+export class SponsorService {
+
+  headers = new Headers({
+    // 'content-type': 'application/json',
+    // 'authorization': 'Token token=9ba5cc744d40feb58d0bc6bce7c8d8be',
```
Comment (bogderp) [inline src/app/services/sponsor.service.ts]: also this token should never be publicly available. This is the dev token so it's fine. but the production token should not be accessible by the public

Comment (HavenTheWanderer) [reply src/app/services/sponsor.service.ts]: The build process takes comments out of the typescript, but that's good to know!

File: src/app/top/top.component.html [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+	<div class="small-8 small-offset-2 column" [@textIn]="">
+		<p style="text-align:center;">January 19-21, 2018 - Michigan State University</p>
+		  <p style="text-align:center;">Registration is coming soon, sign up for email alerts below!</p>
+	</div>
+</div>
+
+<div class="row">
+    <section id="mail">
+    	<!-- MailChimp Sign Up -->
+        <div id="mc_embed_signup" class="small-8 small-offset-2 subscriber" (click)="OnSignUpClick()" [@mailButtonIn]="">
```
Comment (bogderp) [inline src/app/top/top.component.html]: i've noticed on the testflasksite this portion is unstyled.

File: src/assets/data/faqs.json [hunk 8]
```diff
@@ -0,0 +1,222 @@
+﻿[
```
Comment (bogderp) [inline src/assets/data/faqs.json]: if you are using the api to get faqs why do you need this

Comment (HavenTheWanderer) [reply src/assets/data/faqs.json]: We don't.

**Comment by bogderp** [comment]
ahh, yash told me otherwise

File: src/index.html [hunk 9]
```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>SpartaHack IV</title>
+  <base href="/">
+  <link href="http://cdn-images.mailchimp.com/embedcode/classic-10_7.css" rel="stylesheet" type="text/css">
```
Comment (bogderp) [inline src/index.html]: i suggest removing this and styling the form yourself

**Review by bogderp — DISMISSED** [review]
it's a solid intro with the logo and snowflakes fading in but everything after that is lacking.

Need more spacing between date and registration.
mailing list needs styling.
faq needs to be styled
  I can't tell what question corresponds to the answer

white is blinding, black was nicer imo, you could see the snowflakes falling more easily.

transparent buttons at the bottom should match the mailing list button. 

mobile styling is super awkward on an iphone 5.

**Review by nathankurt — APPROVED** [review]

**Pull request merged**